library/profiler_builtins/Cargo.toml

```diff
@@ -9,7 +9,6 @@ bench = false
 doc = false
 
 [dependencies]
-core = { path = "../core" }
 compiler_builtins = { version = "0.1.0", features = ['rustc-dep-of-std'] }
```

Comment:
Shouldn't this dependency be removed too? compiler_builtins pulls in libcore again.

Reply:
I missed this, done.